src/nvim/screen.c

```diff
@@ -5278,7 +5278,7 @@ void screen_puts_len(char_u *text, int textlen, int row, int col, int attr)
   int force_redraw_next = FALSE;
   int need_redraw;
 
-  const int l_has_mbyte = has_mbyte;
+  const bool l_has_mbyte = has_mbyte;
```

Comment:
Sure. This is anecdotal evidence of course, but given the complete lack of complaints like "why doesn't nvim work in my latin1 terminal anymore??" (well it still does, but you need to explicitly configure it now) or "why doesn't [some python or remote plugin] work with `enc=latin1`?" I think we should move forward.